### PR TITLE
Reviewer updates

### PR DESCRIPTION
**Pull request by aaristov**

### CODE REVIEW

File: griottes/graphmaker/graph_generation_func.py [hunk 1]
```diff
@@ -297,7 +304,7 @@ def generate_delaunay_graph(
         else:
             G.add_node(cell)
 
-    pos = {int(i): (cells[i]["z"], cells[i]["x"], cells[i]["y"]) for i in cells.keys()}
+    pos = {int(i): (cells[i]["x"], cells[i]["y"], cells[i]["z"]) for i in cells.keys()}
```
Comment (aaristov) [inline griottes/graphmaker/graph_generation_func.py:307]: I also suggest to follow ZYX order, like the np.array

File: griottes/graphmaker/graph_generation_func.py [hunk 2]
```diff
@@ -16,7 +16,7 @@
 def generate_geometric_graph(
     user_entry,
     descriptors: list = [],
-    dCells: float = 60,
+    cell_cell_distance: float = 60,
```
[the remark this answers is not in the document]
Comment (gronteix) [reply griottes/graphmaker/graph_generation_func.py:19]: Done

File: griottes/graphmaker/graph_generation_func.py [hunk 3]
```diff
@@ -80,14 +80,21 @@ def generate_geometric_graph(
 
     cells = spheroid["cells"]
 
-    # Generate a dict of positions
-    pos = {int(i): (cells[i]["x"], cells[i]["y"], cells[i]["z"]) for i in cells.keys()}
+    if "z" in prop.columns:
```
[the remark this answers is not in the document]
Comment (gronteix) [reply griottes/graphmaker/graph_generation_func.py:83]: good catch

File: griottes/graphmaker/graph_generation_func.py (hunk 1 above)
Comment (gronteix) [reply griottes/graphmaker/graph_generation_func.py]: Done

**Pull request merged**